### ISSUE TITLE
Time lag in appearance of refresh form banner in Enketo Express

### ISSUE THREAD

**Issue by cagulas** [onaio/enketo-express#11]
In the new Enketo Express on zebra.ona.io, the appearance of the banner that alerts the user to refresh the form is seriously delayed.  Users are accustomed to seeing this within a few seconds of opening the form, but in this case, it was more like a few minutes after I opened the form.  If the user refreshes the page, they would lose any data they already entered.

![screen shot 2015-09-11 at 11 38 17 am](https://cloud.githubusercontent.com/assets/3488986/9828911/e61674e6-58c2-11e5-8a02-dfc72e57e116.png)

Also, on Enketo legacy (or whatever is on beta.ona.io right now), the word "Refresh" in the banner is clickable and refreshes the page, but in this instance, you can't click on the word "Refresh" in the banner.

Replaces https://github.com/onaio/core/issues/567

**Comment by cagulas** [comment]
@ukanga this seems to still be an issue.  I replaced a form, but the new form wouldn't refresh.  Even after 5 minutes, still no notification that there was an updated form.  Finally, I had to clear my browser for the new form version to show up.  It's interesting to note that this happens when you replace a form, but also when you delete the form and upload it again.  Sometime, I delete the form and re-upload it, because I don't want to wait for the refresh banner to appear, but now even when I do this, I still see the old form version and have to wait for it to refresh.
